CMakeLists.txt

```diff
@@ -192,33 +186,34 @@ if (USE_LIBCUDF)
     #       instead of hardcoding paths to includes and to libraries, like below
     target_include_directories(meta-cudf-parser-1 PUBLIC
                                /usr/local/include
-                               /usr/local/include/libcudf/libcudacxx
```

Comment:
couldn't find libcudacxx in any folder related to current libraries and deleting this doesn't change anything